options/locale/locale_en-US.ini

```
@@ -899,12 +899,12 @@ migrate.migrate = Migrate From %s
 migrate.migrating = Migrating from <b>%s</b> ...
 migrate.migrating_failed = Migrating from <b>%s</b> failed.
 migrate.migrating_failed.error = Error: %s
-migrate.github.description = Migrating data from Github.com or Github Enterprise.
-migrate.git.description = Migrating or Mirroring git data from Git services
-migrate.gitlab.description = Migrating data from GitLab.com or Self-Hosted gitlab server.
-migrate.gitea.description = Migrating data from Gitea.com or Self-Hosted Gitea server.
-migrate.gogs.description = Migrating data from notabug.org or other Self-Hosted Gogs server.
-migrate.onedev.description = Migrating data from code.onedev.io or Self-Hosted OneDev server.
+migrate.github.description = Migrate data from github.com or other Github instances.
+migrate.git.description = Migrate data from any Git service.
```

```suggestion
migrate.git.description = Migrate a repository only from any Git service.
```
I think with git migrator it's worth pointing out that it targets different set of entities